topics/compute/openstack/docs/getting-started/creating-first-infrastructure.md

```diff
@@ -59,8 +59,40 @@ __5.__ Wait to be redirected back to our dashboard.
 All virtual machine instances running in the cloud have to be accessed remotely. The most common way of accessing
 an instance remotely is SSH. Using SSH requires a pair of keys - a public key and a private key.
 
+=== "CLI"
+
+    !!! note
+
+        Prefer using this method if possible. It is the most secure one, since your private key is kept
+        on your local machine at all times.
+
+
+    You can use the **ssh-keygen** command to create a new pair of private and public keys:
+    ```
+    ssh-keygen -b 4096
```

Comment:
```suggestion
    ssh-keygen -t rsa -b 4096 -f ~/.ssh/id_rsa_cloud"
```

Reply:
when specifying keypair bit length you need to be explicit about type as well.
We also likely want to show case when users are creating specific cloud SSh keypair (that's why I'm adding -f ...)